code/modules/paperwork/faxmachine.dm

```diff
@@ -1,6 +1,7 @@
 var/list/obj/machinery/photocopier/faxmachine/allfaxes = list()
 var/list/admin_departments = list("Central Command", "Sol Government")
 var/list/alldepartments = list()
+var/broadcast_departments = "Stationwide broadcast (WARNING)"
```

Comment:
Does this have to be a global var? It could either be a class constant or a proc static constant.